### DISCUSSION TITLE
Adapt date & time appearance

### DISCUSSION THREAD

**Discussion by cgarciaq** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#545] [Q&A]
I am showing the duration of the last clean in a tile. I would like to process it to be shorter, but I do not see how to process the time (I see there is precision and multiplier, but they do not apply for dates).

Is there a way to process dates & times? I would like, for instance to apply 'relative_time' to the date, which would give me a user-friendly version of it:
```
relative_time converts datetime object to its human-friendly “age” string. The age can be in second,
minute, hour, day, month or year (but only the biggest unit is considered, 
e.g., if it’s 2 days and 3 hours, “2 days” will be returned). Note that it only works for dates in the past.
```
```
  - tile_id: last_clean
    entity: sensor.topoyiyo_last_clean_end
    precision: relative_time
    label: Última limpieza
    tooltip: Fecha de la última limpieza
```

![imagen](https://user-images.githubusercontent.com/6533487/217388885-f340cc18-ef7a-4570-a497-8cace1a825a8.png)

**Comment by PiotrMachowski** [comment]
Unfortunately at this moment you have to create a template sensor to achieve it

**Reply by cgarciaq** [reply]
Thanks for the super-fast answer :-)
